lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -33,6 +36,67 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     end
   end
 
+  def get_token(address) do
+    t1 = Task.async(fn -> TransactionChain.fetch_genesis_address_remotely(address) end)
+    t2 = Task.async(fn -> get_transaction_content(address) end)
+
+    with {:ok, {:ok, genesis_address}} <- Task.yield(t1),
+         {:ok, {:ok, transaction_content}} <- Task.yield(t2) do
+      {:ok,
+       %{
+         genesis: genesis_address,
+         name: Map.get(transaction_content, "name"),
+         supply: Map.get(transaction_content, "supply"),
+         symbol: Map.get(transaction_content, "symbol"),
+         type: Map.get(transaction_content, "type"),
+         properties: do_reduce_properties(Map.get(transaction_content, "properties"))
+       }}
+    else
+      {:ok, {:error, :network_issue}} ->
+        {:error, "Network issue"}
+
+      {:ok, {:error, :decode_error}} ->
+        {:error, "Error in decoding transaction"}
+
+      {:ok, {:error, :invalid_transaction}} ->
```

Comment:
This one is not used right ?
If you want to return this error, you should find a case where this will be happen

Reply:
Which one? :invalid_transaciton one

Reply:
yes

Reply:
Yup there's no case for that one, removing it then.